### DISCUSSION TITLE
Get command options inside code

### DISCUSSION THREAD

**Discussion by RamazanBiyik77** [deployphp/deployer#2298] [Help needed]
I am noob at coding. I need to get the options on dep commands.
i defined a new task inside code. I need to get that option inside "set" task.
For example: `dep my_command server --option project_url="/xxxx/XXXXX"   -v `
i need to get option from code side.

I tried https://www.php.net/manual/en/function.getopt.php but it returns an empty array.

**Comment by antonmedv** [comment]
get(project_url)

**Reply by RamazanBiyik77** [reply]
I need to check if there is option or not.

**Reply by antonmedv** [reply]
has
